butane_cli/src/main.rs

```diff
@@ -20,6 +21,16 @@ struct Cli {
 enum Commands {
     /// Initialize the database.
     Init(InitCommand),
+    /// Add a backend to existing migrations.
+    AddBackend {
```

Comment:
I think I want to change this again, so it is `butane backends [add|remove|list]` reducing this functionality to a single top level sub-command